mmseg/datasets/pipelines/test_time_aug.py

```diff
@@ -95,6 +102,11 @@ def __call__(self, results):
         """
 
         aug_data = []
+        if self.img_scale is None and mmcv.is_list_of(self.img_ratios, float):
+            img_scale = results['img'].shape[:2]
+            self.img_scale = [(int(img_scale[0] * ratio),
+                               int(img_scale[1] * ratio))
+                              for ratio in self.img_ratios]
```

Comment:
`self.img_scale` is set after first `__call__`, so for the second image, it won't set again.

Reply:
fixed it and please check it again!